app/controllers/application_controller.rb

```diff
@@ -109,7 +109,10 @@ def flow_sync_order
         @order.finalize!
         @flow_render = { redirect_to: '/'}
       else
-        @flow_render =  { text: 'Flow error: %s' % @flow_order.error }
+        # @flow_session.use_flow = false
```

Comment:
Likely not clear from our documentation, but this code should do something like
```
if error.code == "order_error"
  .. handle order error code ..
else
  ..
end
```

then you can dispatch appropriately where the order error code = "order_item_not_available"

See: https://docs.flow.io/type/order-error